### ISSUE TITLE
Incorrect error message with String Literal Types

### ISSUE THREAD

**Issue by yves-dolce** [Microsoft/TypeScript#7530]
**TypeScript Version:** 

1.8.5

**Code**

```ts
declare type ConstrainedString = "La première" | "La primera" | "The first";

function f(s : ConstrainedString) {
    console.log(s);
}

f("La primera");
f("La première");
f("The first");


f("De eerste"); // Error message is here, as expected.
```
**Expected behavior:**

The second sentence of the error message I see under Visual Studio Code doesn't seem to make sense, does it?
```Argument of type '"De eerste"' is not assignable to parameter or type '"La première" | "La primera" | "The first"'. Type '"De eerste"' is not assignable to type "The first"'```

I wonder if I should not rather open this under VS Code.

**Actual behavior:** 
Removing that second sentence?

**Comment by DanielRosenwasser** [comment]
Duplicate of #6541. Thanks for filing!

**Comment by DanielRosenwasser** [comment]
Actually, thinking about it more, I think we can potentially make some improvements independent of #6541.

**Comment by DanielRosenwasser** [comment]
Fixed - thanks for suggesting that @yves-dolce! The fix will be in our nightly builds `"tomorrow" | "mañana" | "demain"`. :smile:

**Comment by yves-dolce** [comment]
Just saw this, while watching Build 2017... :-)
Thanks for the follow up.
